packages/integration-tests/projects/suite-web/tests/dashboard/New-account-message.ts

```diff
@@ -0,0 +1,47 @@
+// @group:suite
+// @retry=2
+
+describe('Go into new accounts', () => {
+        beforeEach(() => {
+             // Launches Trezor emulator
+            cy.task('startEmu', { wipe: true });
+            //Initializes Trezor (random seed)
+            cy.task('setupEmu', { needs_backup: false });
+            cy.task('startBridge');
+            cy.viewport(1024, 768).resetDb();
+            cy.prefixedVisit('/');
+            cy.passThroughInitialRun();
+            cy.discoveryShouldFinish();
+        });
+    
+
+    it('New Accounts', () => {
+        //Goes to accounts and verifies that the “New default Accounts natively..” is displayed:
```

Comment:
- Space after //
- Please replace `“` with `"`. I know it is silly but it is common to use `"` instead

Reply:
534889de5464e01a3eb5970394719506dc079b1e done